### DISCUSSION TITLE
求助 Console 适配器无法正常打开终端

### DISCUSSION THREAD

**Discussion by SkilledWhiteMagician** [nonebot/discussions#548] [问答] [answered]
如题初学nonebot，按照 https://nonebot.dev/docs/quick-start 上的快速上手内容尝试建立交互终端

我在两台电脑上都做了相同操作，一台成功运行并可以响应/echo，另一台启动后立刻关闭，提示console适配器被卸载

> Unmount() >>> Frontend(title='Nonebot', classes={'-dark-mode'}, pseudo_classes={'dark', 'focus'}) method=<Frontend.on_unmount>
12-26 05:18:11 [SUCCESS] nonebot | Console unmounted.

完整日志：
> C:\Users\Ruby\Desktop\test\t1>nb run
使用 Python: C:\Users\Ruby\Desktop\test\t1\.venv\Scripts\python.exe
12-26 05:18:10 [SUCCESS] nonebot | NoneBot is initializing...
12-26 05:18:10 [INFO] nonebot | Current Env: prod
12-26 05:18:10 [DEBUG] nonebot | Loaded Config: {'driver': '~fastapi', 'host': IPv4Address('127.0.0.1'), 'port': 8080, 'log_level': 'DEBUG', 'api_timeout': 30.0, 'superusers': set(), 'nickname': set(), 'command_start': {'/'}, 'command_sep': {'.'}, 'session_expire_timeout': datetime.timedelta(seconds=120)}
12-26 05:18:10 [DEBUG] nonebot | Succeeded to load adapter "Console"
12-26 05:18:10 [SUCCESS] nonebot | Succeeded to load plugin "echo" from "nonebot.plugins.echo"
12-26 05:18:10 [SUCCESS] nonebot | Running NoneBot...
12-26 05:18:10 [SUCCESS] nonebot | Loaded adapters: Console
12-26 05:18:10 [INFO] uvicorn | Started server process [14628]
12-26 05:18:10 [INFO] uvicorn | Waiting for application startup.
Unmount() >>> Frontend(title='Nonebot', classes={'-dark-mode'}, pseudo_classes={'dark', 'focus'}) method=<Frontend.on_unmount>
12-26 05:18:11 [SUCCESS] nonebot | Console unmounted.
12-26 05:18:11 [WARNING] nonebot | Press Ctrl-C for Application exit
12-26 05:18:11 [INFO] uvicorn | Application shutdown complete.

之后尝试在这台电脑上卸载重装了所有nonebot相关内容都没有解决，求助各位可能是什么问题？

**Comment by yanyongyu** [comment, marked as answer]
应该是fastapi驱动器用的uvicorn导致的，你可以使用 ~none 启动试试

**Reply by SkilledWhiteMagician** [reply]
非常感谢！

**Comment by qingzhixing** [comment]
有点没看懂,用None启动是怎么启动?
我删除了全部驱动器安装选择None,运行提示我要下载FastAPI,小白求助QAQ
![image](https://github.com/user-attachments/assets/a3ba2cc7-aa9e-49ef-87a0-c9ef29ec64cc)

**Comment by qingzhixing** [comment]
之后我跟着这篇教程跑起来了: https://blog.csdn.net/2402_86265887/article/details/145268763

**Comment by qingzhixing** [comment]
加群问群友了，是要更改配置文件
`DRIVER=~none`
改完之后就跑起来了
![3874aaec90d49db533eae1e0fee6cff6](https://github.com/user-attachments/assets/a56041eb-edd9-409c-9963-ababa0e7a637)
